.github/workflows/mt.yaml

```diff
@@ -13,7 +13,6 @@
 name: Mutation Testing
 
 on:
-  pull_request:
```

Comment:
since we no longer run this job per pull request, the mandatory check for "Mutation Testing Status" needs to be disabled in the repo settings.

this means we can also drop the "meta-job" in this file starting from line 90, if you want.

Reply:
we still run it after merge on the target branch.

Reply:
If we still run it on the main/master branch, that's fine by me :+1: